### PR TITLE
[show] Concurrent test runner test_runner_async.py

### PR DESCRIPTION
**Pull request by rtxm**

### CODE REVIEW

File: examples/recognition/test_runner_async.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
+import asyncio
```
Comment (rtxm) [inline examples/recognition/test_runner_async.py:1]: Start by inspecting the entry point at the end of the file.

**Comment by rtxm** [comment]
For the moment, the output during the run is messy, but the end report is right.

File: examples/recognition/test_runner_async.py [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+completion_cause = "Success"
+
+[expected.interpretation]
+# The URI (without the query string) of the grammar that should match
+type = "builtin:speech/spelling/mixed"
+# The interpretation value we should get
+value = "le137137866cn"
+"""
+    parser = argparse.ArgumentParser(
+        description="Run recogntion tests in .test files",
```
Comment (merwan) [inline examples/recognition/test_runner_async.py:309]:
```suggestion
        description="Run recognition tests in .test files",
```

**Review by merwan — APPROVED** [review]
I just had a quick look, looks good to me. Async/await makes sense for a parallel test runner

**Pull request merged**